src/lib/sw_engine/tvgSwRaster.cpp

```diff
@@ -213,14 +213,42 @@ static bool _translucentRle(SwSurface* surface, const SwRleData* rle, uint32_t c
     auto span = rle->spans;
     uint32_t src;
 
+#if defined(THORVG_NEON_VECTOR_SUPPORT)
+    uint8x8_t vSrc;
```

Comment:
is needed?

Reply:
for (uint32_t x = 0; x < iterations; x+=2) {
            vDst[x] = vadd_u8(vSrc, ALPHA_BLEND_NEON(vDst[x], vIalpha));
        }

Reply:
Sure. I mean [line 233](https://github.com/Samsung/thorvg/blob/5519d4742acaa564043f32bb8016c62598fdaa85/src/lib/sw_engine/tvgSwRaster.cpp#L233) redeclares it

Reply:
ah agree. Sorry I missed it